src/manifest.c

```diff
@@ -594,8 +617,15 @@ struct manifest *load_mom(int version)
 
 	string_or_die(&filename, "%s/%i/Manifest.MoM", state_dir, version);
 	string_or_die(&url, "%s/%i/Manifest.MoM", content_url, version);
+verify_mom:
```

Comment:
Does this label need to be moved before `retrieve_manifests` instead? Else, I'm not seeing how the Manifest.MoM would be redownloaded, only the Manifest.MoM.sig.